config/deployments/integration_test.toml

```
@@ -99,6 +99,7 @@ powertranz.base_url = "https://staging.ptranz.com/api/"
 prophetpay.base_url = "https://ccm-thirdparty.cps.golf/"
 rapyd.base_url = "https://sandboxapi.rapyd.net"
 razorpay.base_url = "https://sandbox.juspay.in/"
+recurly.base_url = "https://$.recurly.com"
```

```suggestion
recurly.base_url = "https://{{merchant_site_name}}.recurly.com"
```
Can you make the same changes in all other places ?